modules/schema/src/lib/schema/classes/type.ts

```diff
@@ -101,9 +101,10 @@ export class Null extends DataType {
     return Type.Null;
   }
   get [Symbol.toStringTag](): string {
-    return 'Null';
+    return `Null`;
```

Comment:
That does not look right, our prettier rules should not use backticks in this case.